OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -51,7 +51,7 @@ public FindResources(Actor self, CPos avoidCell)
 
 		public override Activity Tick(Actor self)
 		{
-			if (IsCanceled || NextActivity != null)
+			if (IsCanceled || NextInQueue != null)
 				return NextActivity;
```

Comment:
Why can't we check `NextActivity != null` in this case?  Is there a situation where `NextActivity` can be null when `NextInQueue` isn't null?